<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newItem</class>
 <widget class="QDialog" name="newItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавити</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#newItem{
border-image: url(:/new/keyboard.jpg) 0 0 0 0 stretch stretch;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color:  rgba(28,22,22,0.6);
border-radius: 5px;
 padding: 2px;
 color:  #fff;
}
</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameofnewitem">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
   border-radius: 5px;
  border: 2px solid  rgba(28,22,22,0.8);
  padding: 2px;
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color:  rgba(28,22,22,0.6);
border-radius: 5px;
 padding: 2px;
 color:  #fff;

}
QLabel:hover{
margin: 0.0px;
}</string>
       </property>
       <property name="text">
        <string>Name of file :/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
   border-radius: 5px;
  border: 2px solid  rgba(28,22,22,0.8);
 padding: 2px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color:  rgba(28,22,22,0.6);
border-radius: 4px;
 padding: 2px;
 color:  #fff;
}
</string>
       </property>
       <property name="text">
        <string>Добавити в: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton{
 background-color:  rgba(28,22,22,0.6);
 border-radius: 4px;
 padding: 2px;
 color:  #fff;
}

</string>
       </property>
       <property name="text">
        <string>C++  </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton{
 background-color:  rgba(28,22,22,0.6);
 border-radius: 4px;
 padding: 2px;
 color:  #fff;
}</string>
       </property>
       <property name="text">
        <string>Qt  </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>29</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 /* background-color: rgb(255, 151, 57); */
border-radius: 5px;
margin:1.5px;
color: #fff;
background-color: #8C46E7;

}
QPushButton:hover  {
background-color: #6E37B6;
margin: 0px;
}
QPushButton:pressed{
background-color: #551E9D;
}</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>29</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 /* background-color: rgb(255, 151, 57); */
border-radius: 5px;
margin:1.5px;
color: #fff;
background-color: #8C46E7;

}
QPushButton:hover  {
background-color: #6E37B6;
margin: 0px;
}
QPushButton:pressed{
background-color: #551E9D;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
